packages/next-app/src/lib/moralis.ts

```diff
@@ -0,0 +1,24 @@
+import { EvmChain } from "@moralisweb3/common-evm-utils";
+import Moralis from "moralis";
+
+Moralis.start({
+  apiKey: process.env.NEXT_PUBLIC_MORALIS_API_KEY as string,
+});
+
+export async function moralisGetTokens({
+  userAddress,
+  chain,
+}: {
+  userAddress: string;
+  chain: "Ethereum" | "Goerli";
+}) {
+  const chainMoralis =
+    chain === "Ethereum" ? EvmChain.ETHEREUM : EvmChain.GOERLI;
+
+  const response = await Moralis.EvmApi.token.getWalletTokenBalances({
+    address: userAddress,
+    chain: chainMoralis,
+  });
+
+  return response.toJSON();
```

Comment:
`#question` does this need to be awaited?

Reply:
No, the response is already awaited, I'm just transforming it in a JSON